x-pack/plugins/fleet/common/services/generate_new_agent_policy.test.ts

```diff
@@ -27,7 +27,6 @@ describe('generateNewAgentPolicyWithDefaults', () => {
       description: 'test description',
       namespace: 'test-namespace',
       monitoring_enabled: ['logs'],
-      is_protected: true,
```

Comment:
should we add this again?

Reply:
we could add with `false`

Reply:
I suppose we could, but I wonder if it makes sense to have a test that changes this since we don't want to generate a new policy with `is_protected` being true since elastic defend is not present in a New Agent Policy.  But I guess if we are just testing functionality, I could? Or maybe I should enforce the code so that the user can't modify `is_protected` as an override prop? What do you think @dasansol92

Reply:
We could test both scenarios, when user sets it to false (it is ok) and when user sets it to true (and we turn it back to false). Also when the prop is not set, and we set it to false.

Reply:
@parkiino you were right, since this is only testing the `generateNewAgentPolicyWithDefaults` and not the API functionality, it's ok to keep it as it is (`is_protected` set to false by default when it is not assigned). @juliaElastic let me know if you think different.